core/modules/layout/includes/block.class.inc

```diff
@@ -136,7 +136,16 @@ class Block extends LayoutHandler {
     if (isset($info['description'])) {
       $preview .= '<p>' . $info['description'] . '</p>';
     }
+    $preview .= $this->getAdminConditionsPreview();
 
+    return $preview;
+  }
+
+  /**
+   * Return a fieldset listing any visibility conditions set for this block.
+   */
+  function getAdminConditionsPreview() {
```

Comment:
This currently conflicts with the PR we made for one field per block. I like your function name `getAdminConditionsPreview` better though, let's keep that instead of `getConditionSummary()`, which is what we have currently.